db/migrate/20170329103706_rename_manual_record_editions_removed_document_ids_to_removed_section_ids.rb

```diff
@@ -0,0 +1,17 @@
+class RenameManualRecordEditionsRemovedDocumentIdsToRemovedSectionIds < Mongoid::Migration
+  def self.up
+    ManualRecord::Edition.collection.update(
```

Comment:
Is it worth defining a `ManualRecord::Edition` class within this migration so that it can run if that class name changes? I don't think the project has been very consistent with that in the past, so I'm not concerned if we don't.